docs/running-on-yarn.md

```diff
@@ -923,7 +927,11 @@ above. For example, you may have configuration like:
 ```properties
   yarn.nodemanager.aux-services = spark_shuffle_x,spark_shuffle_y
   yarn.nodemanager.aux-services.spark_shuffle_x.classpath = /path/to/spark-x-yarn-shuffle.jar,/path/to/spark-x-config
+  # workaround for YARN-11053
+  yarn.nodemanager.aux-services.spark_shuffle_x.system-classes = classes.other.than.spark.shuffle.service.classes
   yarn.nodemanager.aux-services.spark_shuffle_y.classpath = /path/to/spark-y-yarn-shuffle.jar,/path/to/spark-y-config
+  # workaround for YARN-11053
+  yarn.nodemanager.aux-services.spark_shuffle_y.system-classes = classes.other.than.spark.shuffle.service.classes
```

Comment:
Rather than setting it to some arbitrary class name, it would be better to set it to an empty string. In this way, the `ApplicationClassLoader` will be configured to use the default system classes ([ref](https://github.com/apache/hadoop/blob/03cfc852791c14fad39db4e5b14104a276c08e59/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/ApplicationClassLoader.java#L94-L97)). This is probably more desirable behavior than completely overriding the system classes config, and is more similar to the changes from YARN-11053.

Reply:
Let's not add this as it is a workaround for older YARN. It does not need to be a part of Spark docs